website/content/en/docs/concepts/disruption.md

```diff
@@ -169,6 +169,8 @@ For Spot interruptions, the NodePool will start a new node as soon as it sees th
 
 {{% alert title="Note" color="primary" %}}
 Karpenter publishes Kubernetes events to the node for all events listed above in addition to __Spot Rebalance Recommendations__. Karpenter does not currently support taint, drain, and terminate logic for Spot Rebalance Recommendations.
```

Comment:
```suggestion
Karpenter publishes Kubernetes events to the node for all events listed above in addition to [__Spot Rebalance Recommendations__](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/rebalance-recommendations.html). Karpenter does not currently support taint, drain, and terminate logic for Spot Rebalance Recommendations.
```

I'd recommend referencing the link at the first place that it's mentioned

Reply:
now i understand what you meant, ill add this linking on all pages.